pages/[slug]/[[...suffix]].js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import dynamic from 'next/dynamic'
+import { useRouter } from 'next/router'
+
+const Article = dynamic(() => import('../../components/Article/Page'), {
+  ssr: true
```

Comment:
We should add a loading state here and for profile too:
```suggestion
  ssr: true,
  loading: () => <Frame><Loader loading /></Frame>
```

```js
import Frame from '../../components/Frame'
import Loader from '../../components/Loader'
```

To test you can delay the import with the following code:
```js
import('../../components/Article/Page').then((mod) => {
  return new Promise(resolve => {
    setTimeout(() => {
      resolve(mod)
    }, 5000)
  })
})
```

In practice this currently rarely happens because next preload the code most of the time. But could happen on slow networks or if the logic changes.